docs/changelog.txt

```diff
@@ -1,3 +1,54 @@
+0.5.0 (2020-04-26)
+------------------------------------------------------------------------
+
+- Feature: Industries now support up to 16 accepted cargos and 16 produced cargos
+- Feature: Houses now support up to 16 accepted cargos
+- Feature: Support for RoadTypes (a.k.a NRT)
+- Feature: Allow use of switches and random switches as procedures, reducing repetition of switches
+- Feature: Increase railtype ID range from 16 to 64, to support 64 railtypes
+- Feature: Railtype flags RAILTYPE_FLAG_ALLOW_90DEG and RAILTYPE_FLAG_DISALLOW_90DEG for 90 degree curves
+- Feature: Railtype flags RAILTYPE_FLAG_HIDDEN and RAILTYPE_FLAG_PRECOMBINED for precombined spriteset type
+- Feature: Add ANIM_TRIGGER_APT_AIRPLANE_LANDS (follows OpenTTD #7182)
+- Feature: Add station var nearby_tile_grfid (0x6A)
+- Feature: PUSH/POP_COLOUR codes for strings
+- Feature: Support industry vars production_rate (0x70) and transported_last_month_pct (0x71)
+- Feature: Debug_print for GRF parameters
+- Feature: nmlc option to enable debug mode for parser
+- Feature: nmlc option to force regeneration of parser tables
+- Feature: Add TTDPatchFlag 256_persistent_registers (0x80)
+- Change: improved colouring of nmlc errors and warnings (where shell/terminal supports this)
+- Change: Make sprite encoder warnings more informative (#89)
+- Fix #36: Global variable traffic_side was inconsistent with global parameter traffic_side
+- Fix #39: Add compatibility with >=pillow-7.0.0
+- Fix #58: Loosen Pillow version constraint
+- Fix #52: time.clock() has been removed in python 3.8
+- Fix #57: Check coherency of GRF parameters limits
+- Fix 014febc: Wrong link to LICENSE file
+- Fix 78cd84c: Restore 'setup.py install' usability on windows
+- Fix: Add missing parameter in industry and airport tiles in RandomAction2
+- Fix: Convert some code flows to context managers (eliminates 'too many open files' error on macOS with larger grfs)
+- Fix: LOAD_TEMP() 'value' attribute is empty
+- Fix: Update various sprite counts
+- Codechange: Visual Studio Editor Script
+- Codechange: Install_requires in setup.py
+- Codechange: pyinstaller spec file
+- Codechange: Allow PLY to generate parsing/lexing tables (performance improvement)
+- Codechange: Avoid duplicating rail/road/tramtype table code
```

Comment:
That code didn't exist in the last release, so not really a change for these purposes.